### PR TITLE
Added Goat Background

### PR DESCRIPTION
**Pull request by ColdAnkles**
I broke my git a bit and needed to rebuild branches.

This adds goat again

### CODE REVIEW

File: homebrew/reddit/goat-background.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	<info>
+		<name>Goat</name>
+		<description>Goat Background</description>
+		<author url="https://www.reddit.com/r/UnearthedArcana/comments/7jnwr5/was_a_goat_for_10_years/">u/_Metabot</author>
+		<update version="0.0.1">
+			<file name="goat-background.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/homebrew/reddit/goat-background.xml" />
+		</update>
+	</info>
+	
+	<element name="Was a Goat for 10 Years" type="Background" source="Reddit" id="ID_BACKGROUND_GOAT">
```
Comment (swdriessen) [inline homebrew/reddit/goat-background.xml]: even though i don't think there will be many goat backgrounds, please change IDs for background/feature to include author/source to make it more unique (`ID_RDDT`)

File: homebrew/reddit/goat-background.xml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+			<ul class="unstyled">
+				<li><strong>Skill Proficiencies:</strong>Athletics, Animal Handling</li>
+				<li><strong>Languages:</strong>One language that you knew 10 years ago</li>
+				<li><strong>Tool Proficiencies:</strong>One tool proficiency that you had 10 years ago</li>
```
Comment (swdriessen) [inline homebrew/reddit/goat-background.xml]: missing tool proficiency

**Comment by swdriessen** [comment]
Oh, and change the filename to background-goat.xml so that when more backgrounds get added the files are sorted by type. :goat:

File: homebrew/reddit/goat-background.xml (hunk 2 above)
Comment (ColdAnkles) [reply homebrew/reddit/goat-background.xml]: How did I miss that...

File: homebrew/reddit/goat-background.xml (hunk 1 above)
Comment (ColdAnkles) [reply homebrew/reddit/goat-background.xml]: Changed to "ID_RDDT_BACKGROUND_GOAT"

**Comment by ColdAnkles** [comment]
also updated the file name

**Comment by swdriessen** [comment]
Thanks

**Pull request merged**